Cargo.toml

```diff
@@ -31,6 +31,17 @@ test-engine-raft-raft-engine = ["engine_test/test-engine-raft-raft-engine"]
 test-engines-rocksdb = ["engine_test/test-engines-rocksdb"]
 test-engines-panic = ["engine_test/test-engines-panic"]
 pprof-fp = ["pprof/frame-pointer"]
+openssl-vendored = [
```

Comment:
missing `etcd-client/tls-openssl-vendored`?

Reply:
I find `etcd-client` is used some dead code in `backup-stream` crate. These dead code and the `etcd-client` are guarded by `metastore-etcd` features and will not be compiled by default.

Reply:
okay, please add a comment

Reply:
Removed by https://github.com/tikv/tikv/pull/15976